### DISCUSSION TITLE
Add support for  Health check

### DISCUSSION THREAD

**Discussion by shreedhar-ww** [NeoSOFT-Technologies/rest-dot-net-core#46] [Ideas]
DB health-check
External service 
Email

**Comment by ManoramaKumari** [comment]
I have implemented Healthcheck in my Project and it's working. I can share the code along with the document.
![image](https://user-images.githubusercontent.com/88475232/128375550-a3b7e41b-a971-42bd-9c6f-b040adb42d06.png)
